user/user.jsonld

```diff
@@ -0,0 +1,8 @@
+{
+    "@context": {
```

Comment:
- you don't use https://github.com/easy-global-market/cud-data-models/blob/main/jsonld-contexts/user.jsonld#L35 as in the source context (cc @junjou)?
- it is also missing the name of the relationship between `UserProfile` and `Alert`

Reply:
In regards to your first point. We could absolutely change the format to have a userInfos property inside UserProfile entity. Would you prefer this?